cloudsim-plus/src/main/java/org/cloudsimplus/faultinjection/HostFaultInjection.java

```diff
@@ -171,30 +170,37 @@
      * The attribute counts how many host failures the simulation had
      */
     private int numberOfHostFaults;
-    
+
     /**
```

Comment:
Change the attribute documentation to (because now only the last failed VM from a broker is recovery from failure):

A map to store the time (in seconds) VM failures took to be recovered, that is when a clone from the last failed VM for a given broker is recovered. Since a broker just creates a VM clone when all its VMs have failed, only at that time the failure is in fact recovered.